ccp/lib/openshift.py

```diff
@@ -202,7 +183,9 @@ def get_build_info(
         print ("Opening URL to details of the build\n{}".format(url))
 
         try:
-            token = get_jenkins_access_token(self.service_account)
+            token = OpenShiftCmdClient().get_sa_token_from_openshift(
```

Comment:
not initializing it in the __inif__ of `BuildInfo` class ? I'd say we do that, initialize in the __init__ and reference via an object.